src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim().length > 0);
+  const styleOBJ = {};
+
+  styles.forEach((style) => {
+    const [key, value] = style.split(':').map((part) => part.trim());
 
+    if (key && value) {
```

Comment:
Good check for both key and value being present before adding them to the object.